src/scripts/settings.js

```diff
@@ -712,6 +717,13 @@ document.addEventListener('DOMContentLoaded', function (e) {
       document.querySelector(".guide > div[data-id='" + e.target.getAttribute("data-id") + "']").style.display = "none";
     });
 
+    Settings.$useCustomFormat.addEventListener('click', function (e) {
+        Settings.toggleSetting(e.target, (localStorage.getItem("useCustomFormat") !== "true"), "toggle-use-custom-format");
```

Comment:
I think it makes the file inconsistent. @sebastianwahn just probably followed the common practice, which you can see from lines 604-635. I changed it but reverted after I saw this.